vuepress/docs/next/tutorials/consume/high-avail-tutorial.md

```diff
@@ -73,66 +73,4 @@ Validating the shared cache can be done in a process similar to the clustered in
 
 ## Configuring and Deploying with Redis
 
-In this section, an Entando App Engine instance is deployed using Redis as a cache for data served by the App Engine. For more information on the cache configuration for the App Engine, see [high availability in an Entando Application](../../docs/consume/high-avail-application.md).
-
-### Deploy Redis to Kubernetes
-
-1. Create the Redis deployment and expose the endpoints:
-
-```sh
-kubectl create deployment redis --image=redis:7
-```
-```sh
-kubectl expose deployment redis --port=6379 --target-port=6379 -n YOUR-NAMESPACE
-```
-2. Install the Redis CLI for your environment per <https://redis.io/topics/rediscli>.
-3. Get the IP for your Redis deployment:
-```sh
-kubectl get service -n YOUR-NAMESPACE
-```
-4. Validate your deployment:
-
-```sh
-redis-cli -h 10.43.99.198 -p 6379 ping
-```
-   * Should respond PONG.
-
-
-```sh
-redis-cli -h 10.43.99.198 -p 6379 incr mycounter
-```
-   * Should increment each time. 
-
-
-## Configure the Implementation
-
-1. Download the `entando-app.yaml` template:
-
-<EntandoCode>curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/{{$site.themeConfig.entando.fixpack.v71}}/dist/ge-1-1-6/samples/entando-app.yaml"</EntandoCode>
-
-2. Add these environment variables to the `EntandoApp` YAML to match your Redis instance. The variables to create are `REDIS_ACTIVE`, `REDIS_SESSION_ACTIVE`, `REDIS_ADDRESS` (e.g. _redis://localhost:6379_), and `REDIS_PASSWORD`.
-```yaml
-data:
-  environmentVariables:
-    - name: REDIS_ACTIVE
-      value: "true"
-    - name: REDIS_ADDRESS
-      value: YOUR-REDIS-URL 
-    - name: REDIS_SESSION_ACTIVE 
-      value: "true"
-    - name: REDIS_PASSWORD
-      valueFrom:
-        secretKeyRef:
-          key: password
-          name: YOUR-REDIS-SECRET-NAME
-          optional: false 
-```
-Both REDIS_ACTIVE and REDIS_SESSION_ACTIVE need to be set to "true" to enable the storage of HTTP sessions. If only REDIS_ACTIVE is set to "true", Redis is used just for the cache.
-
->NOTE: This example uses a Secret for the `REDIS_PASSWORD`, which is recommended. You can also hardcode the password in the YAML for testing purposes, but the use of clear text passwords in deployment files is not recommended. **Create and use a Secret for the password as a best practice.**
-
-3. Deploy your file
-```sh
-kubectl apply -f entando-app.yaml
-```
-You now have a high availability cluster of Entando with Redis implementation.
\ No newline at end of file
+To employ Redis for cache management, refer to the [Redis Integration tutorial](redis.md).
```

Comment:
@jyunmitch @avdev4j I think we should move this to the prerequisites or make it the first step in the clustering proceed. Logically speaking you should setup Redis first, then increase the number of replicas, not the other way around.